WcaOnRails/app/views/regulations/scrambles/index.html.erb

```diff
@@ -10,7 +10,7 @@
       <strong><%= link_to latest_jarfilename, "./tnoodle/#{latest_jarfilename}" %></strong>
     </span>
     <br/>
-    Last official change: September 25, 2017
+    Last official change: January 25, 2018
```

Comment:
I'd assume we want the last date the release changed, which would be January 29th in this case, right?